### ISSUE TITLE
Key Binding

### ISSUE THREAD

**Issue by jmussman** [peterflynn/brackets-jump-to-matching#2]
Two issues with the key-binding: on the Mac Ctrl-Shift-[ is already mapped in the navigation menu. And the binding syntax doesn't seem to work in the current version of Brackets 1.0.0-15191. I decided that perhaps the closing bracket would be a little more appropriate for matching the "other" brace/bracket/parenthesis so I changed the last line to: 

menu.addMenuItem(COMMAND_ID, [ { key: "Ctrl-Alt-]" } ], Menus.LAST_IN_SECTION, Menus.MenuSection.NAVIGATE_GOTO_COMMANDS);

I just couldn't see creating a branch for this, but it would be nice if a fix was backed into the next iteration of this extension. It doesn't have to be my choice, but a binding that works would be nice :)

**Comment by redmunds** [comment]
I'm hitting this, too:

```
Cannot assign Shift-Cmd-[ to pflynn.jump-to-matching-brace. It is already assigned to navigate.prevDocErrorNotification.js:125 window.console.error
```

@jmussman You can use Debug > Open User Key Map to define your own shortcut so you don't need to update extension locally.

**Comment by peterflynn** [comment]
Fixed in 1.1 thanks to PR #3
